tasks/linter.py

```diff
@@ -52,7 +52,7 @@ def python(ctx):
 
     print(
         f"""Remember to set up pre-commit to lint your files before committing:
-    https://github.com/DataDog/datadog-agent/blob/{DEFAULT_BRANCH}/docs/dev/agent_dev_env.md#pre-commit-hooks"""
+    https://github.com/DataDog/datadog-agent/blob/{get_default_branch()}/docs/dev/agent_dev_env.md#pre-commit-hooks"""
```

Comment:
thought: would we want to use the updated pre-commit hooks even for Agent 6 or would that cause more changes than we can manage?

Reply:
True but this will probably not be `6.53.x` in practice (this function is used to replace `DEFAULT_BRANCH` and we don't plan to add an `agent_context` for python linting)